projects/saturn/src/test/java/io/fairspace/saturn/services/metadata/MetadataPermissionsTest.java

```diff
@@ -0,0 +1,360 @@
+package io.fairspace.saturn.services.metadata;
+
+import io.fairspace.saturn.services.users.User;
+import io.fairspace.saturn.services.users.UserService;
+import io.fairspace.saturn.services.workspaces.Workspace;
+import io.fairspace.saturn.services.workspaces.WorkspaceService;
+import io.fairspace.saturn.vocabulary.FS;
+import io.fairspace.saturn.webdav.Access;
+import io.fairspace.saturn.webdav.DavFactory;
+import org.apache.jena.graph.Node;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.vocabulary.RDF;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class MetadataPermissionsTest {
```

Comment:
Really nice test coverage!
It looks like the tests around workspaces and dav resources access might be duplicating part of DavFactoryAccessTest, but we can consider these more unit tests.

Reply:
thanks! this is truly unit tests